water-hammer/set-case.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
```

Comment:
As a good practice, exit when there are errors or undefined variables:

```suggestion
set -e -u

```